examples/stock_example_parallel.jl

```diff
@@ -0,0 +1,97 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# Solving an decision hazard stock problem using multiprocessed dynamic programming :
+# We decide which quantity to store before the randomness happens
+# If too much energy comes, the excess is wasted
+# If not enough energy comes, we lower accordingly what was decided to discharge
+# Min   E [\sum_{t=1}^TF c_t C_t(u_t)]
+# s.t.    s_{t+1} = s_t - u_t + xi_t, if 0 <= s_t - u_t + xi_t <= s_{max}
+#         s_{t+1} = s_{max}, if s_t - u_t + xi_t > s_{max}
+#         s_{t+1} = 0, if s_t - u_t + xi_t < 0
+#         C_t(u_t) = u_t, if 0 <= s_t - u_t + xi_t
+#         C_t(u_t) = -(0 - s_{t} - xi_t), otherwise
+#         s_0 given
+#         0 <= s_t <= 1
+#         u_min <= u_t <= u_max
+#         u_t choosen knowing xi_1 .. xi_t
+#############################################################################
+
+import StochDynamicProgramming, Distributions
+println("library loaded")
+
+@everywhere begin
```

Comment:
To answer the question above regarding the type of V (I cannot answer directly below I don't know why maybe you commented the PR then commented a diff after). V is now a SharedArray because it is shared by multiple processes